rpm/assets/modules.yaml

```diff
@@ -105,6 +123,13 @@ data:
   license:
     module:
       - MIT
+  dependencies:
+  - buildrequires:
+      walrus: [-5.21]
+      platform: [f28]
+    requires:
+      walrus: [-5.21]
+      platform: [f28]
```

Comment:
This modules requires platform F28 and `walrus` module of any version except `5.21`